src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/ObserveWorldtubeSolution.hpp

```diff
@@ -78,18 +91,20 @@ struct ObserveWorldtubeSolution {
             db::get<Stf::Tags::StfTensor<::Tags::dt<Tags::PsiWorldtube>, 1, Dim,
                                          Frame::Inertial>>(box);
         for (size_t i = 0; i < Dim; ++i) {
-          psi_coefs[1 + i] = psi_dipole.get(i);
-          psi_coefs[num_coefs + 1 + i] = dt_psi_dipole.get(i);
+          psi_coefs[1 + i + pos_offset] = psi_dipole.get(i);
+          psi_coefs[num_coefs + 1 + i + pos_offset] = dt_psi_dipole.get(i);
         }
       }
       const auto legend = [&expansion_order]() -> std::vector<std::string> {
         switch (expansion_order) {
           case (0):
-            return {"Time", "Psi0", "dtPsi0"};
+            return {"Time", "Posx", "Posy", "Posz", "Velx", "Vely",
+                    "Velz", "Accx", "Accy", "Accz", "Psi0", "dtPsi0"};
```

Comment:
To keep this consistent with our other output of 3-vectors, can you do `Position_{x,y,z}`, `Velocity_{x,y,z}`, and `Acceleration_{x,y,z}`?